tests/integration/test_validate_checkpoint.py

```diff
@@ -84,3 +84,58 @@ def test_validate_checkpoint_with_cloud(
         result = validate_cloud_checkpoint.execute(context={})
 
         assert result["success"] is True
+
+    def test_postgres(
+        self,
+        table_name: str,
+        load_postgres_data: Callable[[list[dict]], None],
+        postgres_connection_string: str,
+    ) -> None:
+        load_postgres_data(
+            [
+                {"name": "Alice", "age": 30},
+                {"name": "Bob", "age": 31},
+            ]
+        )
+
+        def configure_checkpoint(context: AbstractDataContext) -> gx.Checkpoint:
+            bd = (
+                context.data_sources.add_postgres(
+                    name=rand_name(),
+                    connection_string=postgres_connection_string,
+                )
+                .add_table_asset(table_name)
```

Comment:
i don't understand how this test is passing - looks to me like you're passing `table_name` in as the asset name, and not providing a `table_name` parameter, which means the table name is defaulting to an empty string. Using kwargs would probably be clearer, but i'm also curious, what data is being validated? If you add an expectation like `ExpectTableRowCountToEqual` does it still pass?

Reply:
Interesting. That was a typo on my part, but it does actually work. https://github.com/great-expectations/great_expectations/blob/develop/great_expectations/datasource/fluent/sql_datasource.py#L1005. Updated both to use keyword args and to assert on table length.

Reply:
🤯